.circleci/run_style_check.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+import argparse
+from git.repo import Repo
+import subprocess
+import os
+import sys
+
+
+def run_black_formatter(apply_diff):
+    """
+    Runs black on committed files for style checks.
+    """
+    # Populate file list from the latest commit.
+    stream = os.popen(
+        "git diff-tree --no-commit-id --name-only --diff-filter=ACM -r HEAD"
+    )
+    files = [
+        y
+        for y in (x.strip() for x in stream.read().splitlines())
+        if y.endswith(".py")
+    ]
+    repo = Repo(os.path.realpath(__file__), search_parent_directories=True)
+    repo_path = repo.git.rev_parse("--show-toplevel")
```

Comment:
Can we use `repo.working_dir` or `repo.working_tree_dir` for this?  https://gitpython.readthedocs.io/en/stable/tutorial.html#meet-the-repo-type mentions `working_tree_dir` several times.

Reply:
Yes, thanks! Done.